ibtk/src/solvers/impls/CCPoissonLevelRelaxationFACOperator.cpp

```diff
@@ -142,20 +142,12 @@ CCPoissonLevelRelaxationFACOperator::CCPoissonLevelRelaxationFACOperator(const s
           CELLG,
           input_db,
           default_options_prefix),
-      d_level_solver_type(),
-      d_level_solvers(),
-      d_level_solver_db(),
-      d_coarse_solver(nullptr),
-      d_coarse_solver_db()
+      d_level_solver_type(CCPoissonSolverManager::PETSC_LEVEL_SOLVER),
+      d_level_solver_default_options_prefix(default_options_prefix + "level_")
```

Comment:
This is a change from what we had before; I assume this fixes something?

Reply:
I moved a line that set a value inside the constructor to the initialization list. This should avoid an extra constructor call, right?